checkstyle/import-control.xml

```diff
@@ -179,6 +179,7 @@
     </subpackage>
 
     <subpackage name="serialization">
+      <allow pkg="org.apache.kafka.clients" />
```

Comment:
Just wondering, what is the reason for this change?

Reply:
Otherwise the build will fail with:
```
> Task :clients:checkstyleMain
[ant:checkstyle] [ERROR] .../clients/src/main/java/org/apache/kafka/common/serialization/ListDeserializer.java:19:1: Disallowed import - org.apache.kafka.clients.CommonClientConfigs. [ImportControl]
```

was addressed in https://github.com/apache/kafka/pull/6592/commits/98e51defc612d7af80b0c81ab6de612520d3ad6b